project-base/app/config/packages/test/prezent_doctrine_translatable.yml

```diff
@@ -0,0 +1,4 @@
+prezent_doctrine_translatable:
+    # do not use fallback locale
+    # (currentLocale of entities should be set by request or stay NULL)
+    fallback_locale: en
```

Comment:
if I get it correctly, should not it be `null` instead of `en` here?

Reply:
text has been updated